README.md

```diff
@@ -1,2 +1,221 @@
-# utilihive-backend-stub-sftp-server
-SFTP backend stub
+# Utilihive SFTP Server Stub
+
+SFTP Server Stub runs an in-memory SFTP server while your tests are running.
```

Comment:
Let's make it clear in the intro that this is a Kotlin project. E.g. ".. is a Kotlin based project that runs an..."

Reply:
Added